server/server.go

```diff
@@ -310,6 +313,13 @@ func (s *Server) startEtcd(ctx context.Context) error {
 		}
 	}
 	s.client = client
+	s.httpClient = &http.Client{
```

Comment:
The client doesn't have a timeout which is different from the original behavior

Reply:
the new timeout is set up by `NewRequestWithContext`. you can check it.

Reply:
like https://github.com/pingcap/pd/pull/2368/files#diff-02ca8f1595ce93b75591586acd227047R1651-R1659

Reply:
Does https://github.com/pingcap/pd/pull/2368/files#diff-91bbeda7eb98a7adc57b9e47e2cf5c2bR1182 also need a context?